src/routes/users/fetchAllUsers.ts

```diff
@@ -0,0 +1,117 @@
+import {
+  RawReplyDefaultExpression,
+  RawRequestDefaultExpression,
+  RawServerDefault,
+  RouteHandler,
+  RouteOptions,
+} from "fastify";
+import { User } from "../../types/User";
+import UserModel from "../../models/user";
+
+type FetchAllUsers = {
+  Reply: User[] | { error: { code: string; message: string } };
+};
+
+const url = "/user";
+
+export const handler: RouteHandler<FetchAllUsers> = async (req, reply) => {
+  try {
+    const users = await UserModel.query().select("firstName", "lastName","areaId");
```

Comment:
in this case, the `select` is not necessary given that we are getting all the columns from the table. so you can leave it as: `await UserModel.query()` and it should work the same way.

Reply:
❗ this suggestion was not applied